src/main/java/ca/corefacility/bioinformatics/irida/ria/web/clients/ClientsController.java

```diff
@@ -148,6 +150,16 @@ public String revokeTokens(@RequestParam Long id) {
 		return "redirect:/clients/" + id;
 	}
 
+	/**
+	 * Revoke access tokens for a specific client.
+	 * @param id - identifier for a client
+	 */
+	@RequestMapping(value = "/ajax/revoke", method = RequestMethod.PUT)
+	public void revokeClientTokens(@RequestParam Long id) {
```

Comment:
So it could, but it would only be creating another function that does the revoking that both call since I do not want the redirect in the new function.  I don't think that is worth it.

Reply:
I just don't like these 2 methods with pretty much the same name doing pretty much the same thing.  I see the comments, but `revokeClientTokens` and `revokeTokens` is a bit too much. 
 Can you at least rename the other one to something like `revokeTokensAndRedirect` so it's more clear why we have the same thing side-by-side?

Also, a `PUT` is bad HTTP grammer here.  It should likely be `DELETE`.

Reply:
Updated in 401c7b79cd22a9e00b3ed61bd3d7f0a71212a567